workflow-language/src/main/resources/swadl-schema-1.0.json

```diff
@@ -2920,6 +2920,10 @@
                         "<mention uid='${event.initiator.user.userId}'/>"
                     ]
                 },
+                {
```

Comment:
can we enforce it to be template? (so we get code completion)

Reply:
if we can address that , otherwise LGTM

Reply:
Done in the next commit